migrations/2024-11-26-115042_add_combined_tables/up.sql

```diff
@@ -0,0 +1,43 @@
+-- Creates combined tables for the following:
+--
+-- Reports: (comment, post, and private_message)
+-- Inbox: (Comment replies, post replies, comment mentions, post mentions, private messages)
+-- Profile: (Posts and Comments)
+-- Modlog: (lots of types)
+-- Search: (community, post, comment, user, url)
+-- TODO not sure about these two:
+-- Home: (comment, post)
+-- Community: (comment, post)
```

Comment:
I dont think the last two make sense, they are also completely separete in the ui.